packages/helium-cli/emissions/mobile.json

```diff
@@ -1,14 +1,14 @@
 [
   {
-    "startUnixTime": 1679064129,
-    "emissionsPerEpoch": "144657534246575"
+    "startTime": "2023-04-18T00:00:00Z",
+    "emissionsPerEpoch": "108,493,150.684932"
```

Comment:
`108,493,150.684932 * (100/66) * 365 = 60B` annualized for the first year as defined in HIP 53 and updated in HIP 75